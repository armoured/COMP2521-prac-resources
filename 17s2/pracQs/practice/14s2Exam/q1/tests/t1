./q1 add m0 m0
