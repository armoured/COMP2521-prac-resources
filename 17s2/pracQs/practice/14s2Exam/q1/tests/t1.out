Matrix X
[     1     2 ]
[     3     4 ]
Matrix Y
[     1     2 ]
[     3     4 ]
add(X,Y)
[     2     4 ]
[     6     8 ]
